doc/specs/#653 - Quake Mode/#653 - Quake Mode.md

```diff
@@ -0,0 +1,629 @@
+        { "keys": "win+2", "monitor": 2, "desktop": "toCurrent" }
+    ]
+}
+```
+
+**Pros**:
+* Is the least verbose way of adding these actions.
+* Don't need to modify (complicate) the current keybinding parsing.
+* Clearly separates these actions from the bulk of other actions.
+
+**Cons**:
+* We're going to need a new _action_ parser just for global hotkeys like this.
+  One that always tries the action as a `globalSummon` action.
+* **H**ard to get these actions into other places in the UI that we want actions
+  (unsure if this is even wanted)
+
+##### Conclusion
+
+From an engineering standpoint, I like 1C the best. It doesn't complicate the
+existing keybinding parsing. The cost of modifying the existing action arg
+parsing just for `globalSummon` actions shouldn't be too high. It's also the
+most ergonomic for users to add this fairly complex setting.
+
+#### Which window, and where?
+
+When looking at the list of requested scenarios, there are lots of different
+ways people would like to use the global summon action. Some want the most
+recent window activated, always. Others want to have one window _per monitor_.
+Some would like to move the window to where the user is currently interacting
+with the PC, and others want to activate the window where it already exists.
+Trying to properly express all these possible configurations is complex. The
+settings should be unambiguous as to what will happen when you press the
+keybinding.
+
+I believe that in order to accurately support all the variations that people
+might want, we'll need two properties in the `globalSummon` action. These
+properties will specify _which_ window we're summoning, and _where_ to summon
+the window. To try and satisfy all these scenarios, I'm proposing the following
+two arguments to the `globalSummon` action:
+
+```json
+"monitor": "any"|"toCurrent"|"onCurrent"|int,
+"desktop": null|"toCurrent"|"onCurrent"
+```
+
+The way these settings can be combined is in a table below. As an overview:
+
+* `monitor`: This controls the monitor that the window will be summoned from/to
+  - `"any"`/omitted: (_default_) Summon the MRU window, regardless of which
+    monitor it's currently on.
+  - `"toCurrent"`: Summon the MRU window TO the current monitor.
+  - `"onCurrent"`: Summon the MRU window ALREADY ON the current monitor.
+  - `int`: Summon the MRU window for the given monitor (as identified by the
+    "Identify" displays feature in the OS settings)
+
+* `desktop`: This controls how the terminal should interact with virtual desktops.
+  - `null`/omitted: (_default_) Leave the window on whatever desktop it's
+    already on - we'll switch to that desktop as we activate the window.
+    - **TODO: FOR DISCUSSION**: Should this just be `any` to match the above?
+        Read the rest of the doc and come back.
+  - `"toCurrent"`: Move the window to the current virtual desktop
+  - `"onCurrent"`: Only summon the window if it's already on the current virtual

```

Comment:
I gotta say "toCurrent" and "onCurrent" looks really alike. Perhaps "floating" & "fixed"? Or something like that. Same for the monitor setting.